client/src/App.tsx

```diff
@@ -36,8 +36,8 @@ const App = () => {
         // Clear interval on unmount
         return function cleanup() {
             clearInterval(timerID);
-        }
-    }, [])
+        };
+    }, []);
```

Comment:
Yeah, you probably want to do the opposite of this lol

If you're using prettier, I believe you can do

```sh
prettier --no-semi
```

or use `prettier-standard`